python-lib/wordcloud_visualizer.py

```diff
@@ -122,7 +116,9 @@ def __init__(
 
     def _color_func(self, word, font_size, position, orientation, random_state=None, **kwargs) -> AnyStr:
```

Comment:
Can you add a lru_cache to this method? It will greatly speed up computation for common words